### ISSUE TITLE
From RSF to SEG-Y

### ISSUE THREAD

**Issue by seaninho94** [ahay/src#221]
Is there a quick way to convert a lot of RSF files into SEGY? I have around 1000 files to convert with a similar name: like shot1,shot2 and so on.

**Comment by sfomel** [comment]
The program is sfsegywrite http://www.ahay.org/wiki/Guide_to_RSF_file_format#Reading_and_writing_SEG-Y_and_SU_files

You can use a Python loop. In SConstruct:

```python
import os, glob
# loop over all files
for rsf in glob.glob('shot*.rsf'):
     # change suffix from 'rsf' to 'segy'
     segy = os.path.splitext(rsf)[0]+'.segy'
    Flow(segy,rsf,'segywrite ....')
```

**Comment by seaninho94** [comment]
I don’t have to use also the command segyheader before segywrite?

**Comment by sfomel** [comment]
Yes, you need to use it if you don't already have trace headers.

**Comment by seaninho94** [comment]
How can I add it to the loop you wrote? I need to do segyheader and then segywrite for all the 1000 files
